libraries/AP_GPS/GPS_Backend.h

```diff
@@ -53,8 +53,8 @@ class AP_GPS_Backend
     virtual void handle_msg(const mavlink_message_t *msg) { return ; }
     virtual void handle_gnss_msg(const AP_GPS::GPS_State &msg) { return ; }
 
-    // driver specific lag
-    virtual float get_lag(void) const { return 0.2f; }
+    // driver specific lag, returns true if the driver is confident in the provided lag
+    virtual bool get_lag(float &lag) const { lag = 0.2f; return true; }
```

Comment:
Should this be GPS_WORST_LAG_SEC? This is going to be used for any GPS driver that doesn't have a specific lag, I'm not sure if for those we should assume the worse or not.